eduNEXT/xblock-filesmanager
# fix: allow to move files to unpublish folder

### Description

When we uploaded files inside a folder and then deleted them, an error occurred. Additionally, when the folders contained subfolders, the files were not moved to the unpublished folder. With this fix, the error disappears, and all files will be correctly moved to the unpublished folder.


BEFORE             |  NOW
:-------------------------:|:-------------------------:
![](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/4e58f8ac-d281-4eb8-ae2c-025e9351a0b1)  |  ![](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/d8a92d7e-3efc-479f-a75a-c608628f9b39)


### How to test it

- Check this branch` jv/fix-delete-folder-content` in the XBlock files manager

1. use node 18
 ```
nvm use
```

2. install dependencies
```
yarn install 
```
3 run to see any change in the react code 
```
yarn dev 

```


**Merge checklist:**
Check off if complete *or* not applicable:
